src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue

```diff
@@ -139,5 +218,18 @@ const handleChangeContextMenuInput = (value: string): void => {
             }
         }
     }
+
+    /* custom design-system component - p-context-menu-item */
+    :deep(.options-menu) {
+
+        .p-context-menu-item {
+            &.selected {
+                @apply bg-white;
+            }
+            &:hover {
+                @apply bg-blue-200;
+            }
+        }
```

Comment:
question: Is it different only in this page? I think we should discuss it with designers..

Reply:
It was decided to keep the style of the existing context menu! thank you!